wagl/reflectance.py

```diff
@@ -786,16 +902,11 @@ def calculate_reflectance(
 
     # lambertian with atmospheric adjacency correction
     if psf_kernel is not None:
-        # NOTES:
-        #    for the time being we're outputting float32 while water atcor details
-        #    are figured out on how sun and sky glint is to be applied.
-        #    it is unknown that if we truncate to int16 now, will this have an
-        #    impact applying glint correction later
-        kwargs["fillvalue"] = numpy.nan
-        kwargs["dtype"] = "float32"
-
-        # update the attrs for the float32 datasets
-        attrs["no_data_value"] = numpy.nan
```

Comment:
We still need this attrs item. Yes it is duplication, but for the time being there is still functions depending on it. Sometime in the future we'll simply just use the `fillvalue` item.

Reply:
attrs["no_data_value"] is same  as previously defined on line 864. So this is simply duplication unless our no_data_value changed.